wcivf/apps/elections/templates/elections/includes/_postal_votes.html

```diff
@@ -0,0 +1,101 @@
+{% load i18n %}
+{% load postcode_tags %}
+{% load humanize %}
+
+<li>
+  <details open class="ds-stack-smaller">
+    <summary>
+      <h2 id='postal-vote'>
+        <span aria-hidden="true"></span>
+        {% blocktrans trimmed %}Voting by post{% endblocktrans %}
+      </h2>
+    </summary>
+
+    {% if card.before_application_deadline %}
+      <p>
+        {% blocktrans trimmed %}
+          If you know that you won't be able to get to the polling station on polling day, you can vote by post.
+        {% endblocktrans %}
+        {% blocktrans trimmed with application_deadline=card.application_deadline|naturalday:"j F Y" %}
+          You have until {{ application_deadline }} to
+        {% endblocktrans %}
+        <a href="https://www.gov.uk/apply-postal-vote">{% trans "Apply for a postal vote" %}</a>.
+      </p>
+      <p>
+        {% blocktrans trimmed %}
+          <a
+            href="https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post"
+          >Read more about applying for a postal vote</a>.
+        {% endblocktrans %}
+        <p>
+    {% else %}
+      <p>
+        {% blocktrans trimmed with application_deadline=card.application_deadline|naturalday:"j F Y" election_date=card.election_date|naturalday:"j F Y" %}
+          The deadline to apply for a postal vote for the elections taking place on {{ election_date }} was {{ application_deadline }}.
+          You can <a href="https://www.gov.uk/apply-postal-vote">apply for a postal vote</a> in future elections.
+        {% endblocktrans %}
+      </p>
+      <p>
+        {% blocktrans trimmed %}
+          <a
+            href="https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/how-vote-post"
+          >Read more about voting by post</a>.
+        {% endblocktrans %}
+        <p>
+    {% endif %}
+
+
+    {% if card.dispatch_dates %}
+      <h3 class="ds-h5">{% trans "If you have a postal vote" %}</h3>
+      <ul>
+        <li>
+          {% blocktrans trimmed with date0=card.dispatch_dates.0|naturalday:"j F Y" date1=card.dispatch_dates.1|naturalday:"j F Y" %}
+            If you already have a postal vote or successfully apply by {{ date0 }}, your pack will be sent from {{ date1 }}.</li>
+          {% endblocktrans %}
+          <li>
+            {% blocktrans trimmed with date1=card.dispatch_dates.1|naturalday:"j F Y" date2=card.dispatch_dates.2|naturalday:"j F Y" application_deadline=card.application_deadline|naturalday:"j F Y" %}
+              If you apply after {{ date0 }} up to 5pm on {{ application_deadline }}, your pack will be sent from {{ date2 }}.
+            {% endblocktrans %}
+          </li>
+        </ul>
+    {% else %}
+      <p>
+        {% blocktrans trimmed %}
+          If you have a postal vote, your local council will send your postal voting pack around three weeks before polling day.
+        {% endblocktrans %}
```

Comment:
On the Electoral Commission site, they asked for

> Your local council will send your postal voting pack after the deadline to become a candidate has passed. This is around three weeks before polling day.

but I changed it to just

> If you have a postal vote, your local council will send your postal voting pack around three weeks before polling day.

here. I think the "deadline to become a candidate" bit is irrelevant detail to most people (excepting those who eagerly await SOPN day each election cycle)